doc/faq.md

```diff
@@ -328,11 +328,11 @@ If your issue is related to crashes, then you need to obtain core file prior to
 
 
 
-For issues about the rules, we usually need a **message sample** that causes a problem. To protect your privacy, you can remove unrelevant headers and content. E.g. you can blind message sender/recipients, subject and/or other fields.
+For issues about the rules, we usually need a **message sample** that causes a problem. To protect your privacy, you can remove irelevant headers and content. E.g. you can blind message sender/recipients, subject and/or other fields.
```

Comment:
I believe the English spelling is "irrelevant".

Reply:
thanks, indeed, I will change this